src/components/Gallery/ImageGrid.tsx

```diff
@@ -0,0 +1,37 @@
+import { FunctionComponent } from 'react';
+import Imgix from 'react-imgix';
+
+import './ImageGrid.css';
+
+interface ImageGridProps {
+  images: string[];
+  onClick: (event: any) => void;
+}
+
+export const ImageGrid: FunctionComponent<ImageGridProps> = ({
```

Comment:
Do we think it's worth being consistent with how each of these components are implemented, ie using only class or functional components across the board? Or is it perfectly fine to mix the two?

Reply:
Don't see an issue with mixing the two. IMO class components make more sense for components who _need_ the react life-cycle methods. Functional components make sense for more presentational components.

Happy to refactor as class comp. if preferred, but my 2c is that func. comps. make more sense for strictly presentational comps. like these.

Although this component does have a "side effect" in that it has an `onClick`. But that will be extracted out to a button in a subsequent commit anyway.

Reply:
> Happy to refactor as class comp. if preferred, but my 2c is that func. comps. make more sense for strictly presentational comps. like these.

Cool, just wanted to check. This explanation makes sense to me 👍